src/views/v2/list/nav/prev.php

```diff
@@ -21,14 +21,12 @@
 		rel="prev"
 		class="tribe-events-c-nav__prev tribe-common-b2 tribe-common-b1--min-medium"
 		data-js="tribe-events-view-link"
-	>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__prev-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
+	><?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 6ffcda5d57682070552f069f03fa244f</sub>